tests/lib/rules/newline-per-chained-call.js

```diff
@@ -116,15 +153,40 @@ ruleTester.run("newline-per-chained-call", rule, {
             "    'method3' :",
             "    'method4']()"
         ].join("\n"),
+        output: [
+            "anObject.method1().method2()\n['method' + n]()\n[aCondition ?",
```

Comment:
Nitpick: Since we're actually writing this out with whitespace I think it would be clearer to add a new element to the array instead of inlining a `\n` character.